models/ht_connection.h

```diff
@@ -200,8 +197,8 @@ template < typename targetidentifierT >
 HTConnection< targetidentifierT >::HTConnection()
   : ConnectionBase()
   , weight_( 1.0 )
-  , tau_P_( 50.0 )
-  , delta_P_( 0.2 )
+  , tau_P_( 500.0 )
+  , delta_P_( 0.5 )
```

Comment:
Since this could potentially have an influence on existing user scripts, could you justify the change in default parameters?